graylog2-web-interface/src/components/users/PermissionSelector.jsx

```diff
@@ -90,67 +105,67 @@ const PermissionSelector = React.createClass({
   /*
    * onClick actions for single edits
    */
-  _toggleStreamReadPermissions(stream) {
-    this._toggleReadPermissions('streams', Immutable.Set.of(stream.id));
+  _setStreamReadPermissions(stream) {
+    this._setReadPermissions('streams', Immutable.Set.of(stream.id));
   },
 
-  _toggleStreamEditPermissions(stream) {
-    this._toggleEditPermissions('streams', Immutable.Set.of(stream.id));
+  _setStreamEditPermissions(stream) {
+    this._setEditPermissions('streams', Immutable.Set.of(stream.id));
   },
 
-  _toggleDashboardReadPermissions(dashboard) {
-    this._toggleReadPermissions('dashboards', Immutable.Set.of(dashboard.id));
+  _setDashboardReadPermissions(dashboard) {
+    this._setReadPermissions('dashboards', Immutable.Set.of(dashboard.id));
   },
 
-  _toggleDashboardEditPermissions(dashboard) {
-    this._toggleEditPermissions('dashboards', Immutable.Set.of(dashboard.id));
+  _setDashboardEditPermissions(dashboard) {
+    this._setEditPermissions('dashboards', Immutable.Set.of(dashboard.id));
```

Comment:
The naming of the single permission change functions was better before the change, because they are not setting an explicit permission, they are toggling a given permission.